NETlib_v0.4.py

```diff
@@ -98,17 +100,17 @@ def getAtmosphere(self, atm = None):
         v: array of frequency
         dv: frequency step
         Tx: array of Transmission coef
-        
+
         """
-        
+
         zones = ['Antarctic','Arctic','tropical','northern_midlatitude','southern_midlatitude']
-        zones_alts = array(range(0,11)+[20,30,40,50])*1000
+        zones_alts = array(list(range(0,11))+[20,30,40,50])*1000
 
         if atm == None:
             atm = {}
 
         if 'atmdir' not in atm.keys():
-            atm['atmdir'] = '/n/home01/dbarkats/work/20170801_S4_NET_forecast_python/am_spectra/'
+            atm['atmdir'] = os.path.join(self.path, 'am_spectra')
```

Comment:
Note that the data directory is now assumed to live beside the path to this file.